From d7ee92e1cb19bc37688bfaea0efb6308febb47b4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Wed, 6 Feb 2019 11:38:49 +0200
Subject: [PATCH] init: fix fbsplash

The configuration option was renamed when parts of our patch
were upstreamed to busybox.
---
 initramfs-init.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/initramfs-init.in b/initramfs-init.in
index 252b9ca..62b041d 100755
--- a/initramfs-init.in
+++ b/initramfs-init.in
@@ -596,7 +596,7 @@ if [ -f "$sysroot/etc/.default_boot_services" -o ! -f "$ovl" ]; then
 fi
 
 if [ "$KOPT_splash" != "no" ]; then
-	echo "IMAGE_ALIGN=CM" > /tmp/fbsplash.cfg
+	echo "IMG_ALIGN=CM" > /tmp/fbsplash.cfg
 	for fbdev in /dev/fb[0-9]; do
 		[ -e "$fbdev" ] || break
 		num="${fbdev#/dev/fb}"
-- 
2.20.1

